CHANGELOG.md

```diff
@@ -19,7 +32,7 @@
 - New `devtool` command: `tag`. This creates a new git tag for the specified
   release number, based on the changelog contents.
 - New doc section about building with glibc.
-- New API call: `PATCH /machine-config/`, used to update VM configuration, 
```

Comment:
This looks like an error. This entry shouldn't be in 0.16.0. It was introduced by d4c14d82db3eb936739819f0bb615eebbd1a5291 which is the first commit after the 0.16.0 release.

Reply:
Uh, I see. The aforementioned commit modifies the wrong section of the Changelog. I moved the changes to the "Added" and "Changed" section release 0.17.0. Take a look.